.github/configs/feature.yaml

```diff
@@ -16,6 +16,21 @@ zkevm:
   fill-params: --from=Cancun --until=Prague --block-gas-limit 36000000 -m zkevm ./tests
   solc: 0.8.21
   feature_only: true
+zkevm_60M:
+  evm-type: zkevm
+  fill-params: --from=Cancun --until=Prague --block-gas-limit 60000000 -m zkevm ./tests
+  solc: 0.8.21
+  feature_only: true
+zkevm_90M:
+  evm-type: zkevm
+  fill-params: --from=Cancun --until=Prague --block-gas-limit 90000000 -m zkevm ./tests
+  solc: 0.8.21
+  feature_only: true
+zkevm_120M:
+  evm-type: zkevm
+  fill-params: --from=Cancun --until=Prague --block-gas-limit 120000000 -m zkevm ./tests
+  solc: 0.8.21
+  feature_only: true
```

Comment:
Are these automatically triggered on any `zkevm` tag?

Reply:
Yes that's correct, the tag is "starts with".